### PR TITLE
Update to 24.9

### PR DESCRIPTION
**Pull request by hmaarrfk**
Closes #658 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
xref: https://github.com/conda-forge/miniforge/issues/656

**Comment by hmaarrfk** [comment]
Lets merge https://github.com/conda-forge/miniforge/pull/657 first

**Comment by jaimergp** [comment]
Rerunning ppc64le 🤞

File: Miniforge3/construct.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% set name = os.environ.get("MINIFORGE_NAME", "Miniforge3") %}
-{% set version = os.environ.get("MINIFORGE_VERSION", "24.7.1-1") %}
-{% set conda_libmamba_solver_version = "24.7.0"%}
+{% set version = os.environ.get("MINIFORGE_VERSION", "24.9.1-1") %}
+{% set conda_libmamba_solver_version = "24.9.0"%}
 # when mamba_version is updated here, also update MICROMAMBA_VERSION
 # in scripts/build.sh
 {% set mamba_version = "1.5.9"%}
```
Comment (jaimergp) [inline Miniforge3/construct.yaml:6]:
```suggestion
{% set mamba_version = "1.5.10"%}
```

Published a couple weeks ago

**Comment by hmaarrfk** [comment]
lets see what happens, at the very least we should release to start to depreciate PyPy

**Comment by hmaarrfk** [comment]
Nice we hit our own brownouts...

File: Miniforge3/construct.yaml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 {% set name = os.environ.get("MINIFORGE_NAME", "Miniforge3") %}
-{% set version = os.environ.get("MINIFORGE_VERSION", "24.7.1-1") %}
-{% set conda_libmamba_solver_version = "24.7.0"%}
-# when mamba_version is updated here, also update MICROMAMBA_VERSION
-# in scripts/build.sh
-{% set mamba_version = "1.5.9"%}
+{% set version = os.environ.get("MINIFORGE_VERSION", "24.9.1-1") %}
+{% set conda_libmamba_solver_version = "24.9.0"%}
+# when mamba_version is updated here, also update MICROMAMBA_VERSION in scripts/build.sh
+{% set mamba_version = "2.0.2"%}
```
Comment (hmaarrfk) [inline Miniforge3/construct.yaml:5]:
```suggestion
{% set mamba_version = "1.5.9"%}
```

File: scripts/build.sh [hunk 3]
```diff
@@ -38,7 +38,7 @@ cp LICENSE "${TEMP_DIR}/"
 ls -al "${TEMP_DIR}"
 
 if [[ "${TARGET_PLATFORM}" != win-* ]]; then
-    MICROMAMBA_VERSION=1.5.9
+    MICROMAMBA_VERSION=2.0.2
```
Comment (hmaarrfk) [inline scripts/build.sh:41]:
```suggestion
    MICROMAMBA_VERSION=1.5.9
```

**Comment by hmaarrfk** [comment]
@conda-forge/miniforge is there a reason why docker might be failing to find the images now?

**Pull request merged**